### ISSUE TITLE
[bug] Specified version of pg_dumpall not found during backup

### ISSUE THREAD

**Issue by nick322** [imagegenius/docker-baseimage-immich#11]
I noticed the log showing the following error message.

```log
[Nest] 261  - 12/11/2024, 2:00:00 AM     LOG [Microservices:BackupService] Database Backup Starting. Database Version: 14
[Nest] 261  - 12/11/2024, 2:00:00 AM   ERROR [Microservices:Error: spawn /usr/lib/postgresql/14/bin/pg_dumpall ENOENT] Backup failed with error
[Nest] 261  - 12/11/2024, 2:00:00 AM   ERROR [Microservices:Error: spawn /usr/lib/postgresql/14/bin/pg_dumpall ENOENT] Database Backup Failure
[Nest] 261  - 12/11/2024, 2:00:00 AM   ERROR [Microservices:BackupService] Gzip exited with code 0 but pgdump exited with -2
```

When I enter the container, only version 16 is available:

```sh

❯ docker exec -it immich /bin/bash  
root@d62299726861:/# ll /usr/lib/postgresql/  
total 0  
drwxr-xr-x 1 root root   4 Dec  8 05:42 ./  
drwxr-xr-x 1 root root 376 Dec  9 10:36 ../  
drwxr-xr-x 1 root root  12 Dec  8 05:42 16/  
```
After reviewing the project's Dockerfile, it seems that the latest version of postgresql-client is installed.
Is it possible to support pg_dumpall for PostgreSQL versions 14 and 15?

**Comment by nick322** [comment]
I checked the most recent CI status:
https://ci.imagegenius.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-immich/detail/cuda/22/pipeline/

I found that the docker-immich Dockerfile runs apt-get update and encounters the following issue:
```
The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7FCC7D46ACCC4CF8
```
This error occurs because the file /etc/apt/sources.list.d/pgdg.list was not deleted, causing this problem.

**Comment by nick322** [comment]
fix: remoge pgdg source list #13
@martabal Thanks.
